src/test/java/CalculateTest.java

```diff
@@ -0,0 +1,228 @@
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import java.util.NoSuchElementException;
+import org.junit.jupiter.api.Test;
+
+class CalculateTest {
+    private static final Calculate CALCULATOR = new Calculate();
+    private double actual;
+    private double answer;

```

Comment:
```suggestion
    private double expected;

```

`answer` less informative name